### ISSUE TITLE
Event tracking opt-out option

### ISSUE THREAD

**Issue by chawyehsu** [RabbyHub/Rabby#1023]
[GA](https://github.com/RabbyHub/Rabby/search?q=reactga) and [own tracker](https://github.com/RabbyHub/Rabby/search?q=stats) included. Removing it completely #1018 while having the data analysis demand is unlikely, but please give the option to opt out.

**Comment by My1** [comment]
I would even say tracking should be opt in

**Comment by chawyehsu** [comment]
Yeah but talking about _business demand_, you have no idea whether even an opt-out option would be provided. Hence, you might have to block the tracking domain from your network layer manually.

**Comment by armaccloud** [comment]
> Yeah but talking about _business demand_, you have no idea whether even an opt-out option would be provided. Hence, you might have to block the tracking domain from your network layer manually.

Sorry if this is an obvious question, but how can I find out the tracking domain?
Also, isn't this stats functionality a violation of their privacy policy, or am I mixing things up?

![image](https://github.com/RabbyHub/Rabby/assets/9272078/70a4556b-bb3e-40b6-8b3d-9997001c550e)

**Comment by My1** [comment]
> Yeah but talking about _business demand_, you have no idea whether even an opt-out option would be provided. Hence, you might have to block the tracking domain from your network layer manually.

I think laws have a LITTLE BIT (obviously /s) more importance over whatever cute "business demand" that may exist
